OutOfSchool/OutOfSchool.DataAccess/Models/Notification.cs

```diff
@@ -11,7 +12,7 @@ public class Notification : IKeyedEntity<Guid>
         [Required]
         public string UserId { get; set; }
 
-        public string Text { get; set; }
+        public Dictionary<string, string> Data { get; set; }
```

Comment:
just a thought: Maybe make the column nullable? Empty dictionary is serialised as empty JSON object `{}` if i remember correctly, taking 8 bytes of space even when the data is empty. null in InnoDB takes way less :)

Reply:
Done